Makefile

```diff
@@ -53,8 +53,12 @@ CFLAGS += -mmacosx-version-min=10.5 \
 		  -fno-builtin
 endif
 
+# on MacOS, compile in Universal format by default
+MACOS_UNIVERSAL ?= yes
```

Comment:
cool, but why dont you put this macro into the block checking for IS_APPLE?
that block is at https://github.com/aquynh/capstone/blob/master/Makefile#L261

Reply:
Fixed. I did this to make it less complicated.